packages/zero-runtime/src/processors/css.ts

```diff
@@ -0,0 +1,179 @@
+import type { Expression } from '@babel/types';
+import { validateParams } from '@linaria/tags';
+import type {
+  CallParam,
+  TemplateParam,
+  Params,
+  TailProcessorParams,
+  ValueCache,
+} from '@linaria/tags';
+import type { Replacements, Rules } from '@linaria/utils';
+import { ValueType } from '@linaria/utils';
+import type { CSSInterpolation } from '@emotion/css';
+import deepMerge from 'lodash.merge';
```

Comment:
@mui/utils has `deepmerge` function. Is there any differences?

Reply:
That has bare-minimum implementation since it gets added to the user bundle. lodash one is more feature complete and will be only used at build time. So I'm using this.